datasetComparison/src/test/resources/json_orig_show.txt

```diff
@@ -0,0 +1,9 @@
++---+------------------------------------------+
+|id |legs                                      |
++---+------------------------------------------+
+|3  |[[[[300, [[[6, 7, 8b, 9, 0c, 1]]]]], 300]]|
```

Comment:
Not an issue, just a note. The formatting of `.show()` can change from version to version. For instance, the Spark devs have introduced spaces between values after comma between Spark `2.2.2` and `2.4.3` (e.g. `1,2,3` -> `1, 2, 3`). This makes such tests fragile.

But this type of representation is so visually convenient, I'd probably not change it. Just keep an eye on possible future changes in `.show()` formatting.